src/content/docs/journey-migration/making-the-switch.mdx

```diff
@@ -0,0 +1,127 @@
+
+Digging deeper into New Relic features will help you do all that and more, and you can continue to explore the platform [for free](https://newrelic.com/signup). One of our core principle is to help you do your best work every day using their data at every phase of development. Once you make the switch, you'll never look at your data the same way again.
+
+## Do more with your data [#do-more]
+
+New Relic has a host of features that enable you to maximize the information you get from your data. Once you have your infrastructure, application, and log monitors set up, you can use any of the following observability features to help you get the most out of your stack.
+
+<CollapserGroup>
+    <Collapser
+        id="synthetics"
+        title="Synthetics"
+    >
+    Create tests using synthetic input to simulate traffic, resolve ability issues, and measure uptime with zero effects on your users. Test what you need when you need it, no matter how specific your needs are. 
+        * Simulated traffic can be public or private across thousands of possible locations.
+        * Find out about sub-optimal performance before your customers have a chance to know about them.
+        * Get full context on your stack to identify specifically where your issues are arising from.
+        * Use our synthetic testing in pre-production environments, or adjust any pages that are already in production.
+
+    Learn more about synthetics in our [short overview](https://newrelic.com/platform/synthetics), or see our [introduction to synthetics](/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring/) documentation.
+    </Collapser>
+    <Collapser
+        id="browser"
+        title="Browser monitoring"
+    >
+    Get complete and total observability for your web pages, whether your site is large or small. Measure your core web vitals and errors to ensure a satisfying customer experience from the first click to the last.
+        * Install browser agents as quickly as you would any other using our guided installations.
+        * Find out what's causing issues with how customers interact with your site from the beginning to the end of a transaction.
+        * Monitor your AJAX requests, JavaScript errors, and more as they occur in real time so you can resolve any issues before they become problems.
+        * Customize your monitoring charts and share them across your organization so everyone has the same context you do.
+    Learn more about browser monitoring in our [short overview](https://newrelic.com/platform/browser-monitoring), or see our [introduction to browser monitoring](/docs/browser/browser-monitoring/getting-started/introduction-browser-monitoring/) documentation.
```

Comment:
Is this meant to be on its own line? In the build it gets combined with the last bullet
![Uploading image.png…]()

Reply:
Should be fixed now, I'll try to keep an eye on it.